src/main.rs

```diff
@@ -34,7 +34,7 @@ const DEFAULT_INSTANCE_ID: &str = "anonymous-instance";
 
 fn main() {
     LOGGER
-        .init(&"", None, None)
+        .init(&"", None, None, vec![])
```

Comment:
Not necessary related to this PR, but I noticed that when we run firecracker with no additional parameter and not in a jailer context, the errors will have a weird format something like [:ERROR:vmm/src/lib.rs]. I see two options here: a) either we remove the ":" when the instance id is not there or b) we insert the `DEFAULT_INSTANCE_ID` which is sent in the shared info upon socket binding. What do you say? Should we open an issue maybe?

Reply:
Yup, it looks strange, and should be an easy fix. Logged #763.